core/src/main/java/com/linecorp/armeria/common/DefaultRequestHeadersBuilder.java

```diff
@@ -133,18 +131,9 @@ public RequestHeadersBuilder authority(String authority) {
     @Override
     public RequestHeadersBuilder acceptLanguages(Iterable<LanguageRange> acceptLanguages) {
         requireNonNull(acceptLanguages, "acceptLanguages");
-        final Collection<LanguageRange> languageRangeCollection;
-        if (acceptLanguages instanceof Collection) {
-            languageRangeCollection = (Collection<LanguageRange>) acceptLanguages;
-        } else {
-            languageRangeCollection = ImmutableList.copyOf(acceptLanguages);
-        }
+        final List<LanguageRange> languageRangeCollection = ImmutableList.copyOf(acceptLanguages);
```

Comment:
Doesn't this create a new list instance unnecessarily? `acceptLanguages()` could accept `Iterable<LanguageRange>` instead?